FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Romeo Flauta on 4/10/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+		var feed: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	//internal representation of FeedImage but for the API module
+	private struct Item: Decodable {
+		let imageId: String
+		let imageDesc: String?
+		let imageLoc: String?
+		let imageUrl: URL
+
+		var item: FeedImage {
+			return FeedImage(id: UUID(uuidString: imageId)!, description: imageDesc, location: imageLoc, url: imageUrl)
```

Comment:
See my previous comment about making `imageId` a `UUID`.

Notice how by using the type system you can avoid force casting the conversion of a `String` to `UUID` resulting in safer code as you avoid possible runtime crashes.

```suggestion
			return FeedImage(id: imageId, description: imageDesc, location: imageLoc, url: imageUrl)
```